docs/api_reference.md

```diff
@@ -1,12 +1,12 @@
-## Overview of the Courtgrabber REST API
+# The Courtgrabber REST API
 
-The following documentation explains the Courtgrabber REST API. If you would like to use the Courtgrabber CLI, please refer to the [user guide](./user_guide.md). 
+The following documentation explains the Courtgrabber REST API. If you would like to use the Courtgrabber CLI, please refer to the [user guide](/user_guide). 
```

Comment:
I thought we agreed that links would be of the style `./user_guide.md` instead of `/user_guide` since the former works with mkdocs and github whereas the latter only works with mkdocs?

Reply:
Sorry, I must have overlooked this one. I've pushed the change. Thanks.